lib/libtrakt/exceptions.py

```diff
@@ -1,8 +1,43 @@
-class traktException(Exception):
-    pass
+from requests.exceptions import (ConnectionError, Timeout, TooManyRedirects)
 
-class traktAuthException(traktException):
-    pass
 
-class traktServerBusy(traktException):
-    pass
+class TraktException(Exception):
+    """A Generic Trakt Exception"""
+
+
+class TraktAuthException(TraktException):
```

Comment:
While the base class makes sense to prefix with Trakt, prefixing the remainder makes it unecessarily verbose.  Verbosity that is easily replicated through import statemtents. e.g.:
```python
import trakt
trakt.exceptions.AuthException

from trakt.exceptions import AuthException as TraktAuthException
```